packages/@okta/vuepress-site/docs/reference/api/event-hooks/index.md

```diff
@@ -97,7 +98,8 @@ curl -v -X POST \
         "items" : [
             "user.lifecycle.create",
             "user.lifecycle.activate"
-        ]
+        ],
+        "filter": null
```

Comment:
Feel free to ignore this, but there are some spacing inconsistencies between colons, and differing indents for some of the "Filter" params.

Reply:
Thanks! Good eye.